.travis/setup_binfmt_misc.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash -ex
+
+echo ':arm:M::\x7fELF\x01\x01\x01\x00\x00\x00\x00\x00\x00\x00\x00\x00\x02\x00\x28\x00:\xff\xff\xff\xff\xff\xff\xff\x00\xff\xff\xff\xff\xff\xff\xff\xff\xfe\xff\xff\xff:/usr/local/bin/qemu-arm-static:' > /proc/sys/fs/binfmt_misc/register
```

Comment:
> On 28 Nov 2015, at 7:52 PM, Bob van der Linden <notifications@github.com> wrote:
> 
> In .travis/setup_binfmt_misc.sh <https://github.com/machinekit/machinekit/pull/801#discussion_r46079625>:
> 
> > @@ -0,0 +1,3 @@
> > +#!/bin/bash -ex
> > +
> > +echo ':arm:M::\x7fELF\x01\x01\x01\x00\x00\x00\x00\x00\x00\x00\x00\x00\x02\x00\x28\x00:\xff\xff\xff\xff\xff\xff\xff\x00\xff\xff\xff\xff\xff\xff\xff\xff\xfe\xff\xff\xff:/usr/local/bin/qemu-arm-static:' > /proc/sys/fs/binfmt_misc/register
> Also, that means I have to re-evaluate drone.io as I didn't realized binfmt_misc wasn't needed at all anymore.
> 

BTW, Drone.io <http://drone.io/> already uses docker. Don’t know if we can run docker inside a docker container :D